source/change-streams/tests/change-streams-errors.yml

```diff
@@ -50,4 +50,27 @@ tests:
           database_name: *database_name
     result:
       error:
-        code: 40324
\ No newline at end of file
+        code: 40324
+  -
+    description: Change Stream should error when _id is projected out
+    minServerVersion: "4.1.11"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline:
+      -
+        $project: { _id: 0 }
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: insertOne
+        arguments:
+          document:
+            z: 3
+    expectations: []
```

Comment:
https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#spec-test-format suggests this field is optional. If we're not going to include any CommandStartedEvents here, I'd suggest just omitting the field entirely lest some test runner interpret this as "no events should be expected".